pkg/utils/utils.go

```diff
@@ -30,3 +30,22 @@ func RandStringBase64(length int) (string, error) {
 
 	return randStringBase64, nil
 }
+
+const letters = "abcdefghijklmnopqrstuvwxyz0123456789"
+
+func RandStringBytes(length int) (string, error) {
```

Comment:
surprised golang or a vendored library does not have the equivalent functionality of `openssl rand -hex N` . . .

Reply:
I have suggested above we maybe consider some hashing instead of using random